pkg/client/command_line.go

```diff
@@ -15,62 +16,215 @@ func AddArmadaApiConnectionCommandlineArgs(rootCmd *cobra.Command) {
 	viper.BindPFlag("armadaUrl", rootCmd.PersistentFlags().Lookup("armadaUrl"))
 }
 
-func LoadCommandlineArgsFromConfigFile(cfgFile string) {
+// LoadCommandlineArgsFromConfigFile load config from at exePath/armadactl-defaults.yaml, where
```

Comment:
This is probably just a style thing, I find this hard to understand.

Here is another attempt, which may not be any better:
// LoadCommandlineArgsFromConfigFile loads armadactl config
// armadactl-defaults.yaml - From exePath, where exePath is the path to the armadactl executable
// armada config file           - From cfgFile or defaulting to $HOME/.armadactl
// These configs are then merged